celerdata/assets/logs/celerdata_tests.yaml

```diff
@@ -0,0 +1,58 @@
+id: "celerdata"
+tests:
+ -
+  sample: "2024-03-28 10:21:53,960 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():282] create built-in role cluster_admin[-3]"
+  result:
+    custom:
+      class_name: "AuthorizationMgr"
+      file_line: 282
+      loglevel: "INFO"
+      method_name: "initBuiltinRoleUnlocked"
+      timestamp: 1711621313960
+    message: "create built-in role cluster_admin[-3]"
+    status: "info"
+    tags:
+     - "source:LOGS_SOURCE"
+    timestamp: 1711621313960
+ -
+  sample: "2024-03-28 18:58:43,380 INFO (nioEventLoopGroup-4-3|154) [RestBaseAction.handleRequest():73] receive http request. url=/api/health"
+  result:
+    custom:
+      class_name: "RestBaseAction"
+      file_line: 73
+      loglevel: "INFO"
+      method_name: "handleRequest"
+      timestamp: 1711652323380
+    message: "receive http request. url=/api/health"
+    status: "info"
+    tags:
+     - "source:LOGS_SOURCE"
+    timestamp: 1711652323380
+ -
+   sample: "2024-04-10 10:32:23.937+08:00 INFO (stateChangeExecutor|95) [StateChangeExecutor.runOneCycle():179] finished to transfer FE type from INIT to LEADER"
+   result:
+     custom:
+       class_name: "StateChangeExecutor"
+       file_line: 179
+       loglevel: "INFO"
+       method_name: "runOneCycle"
+       timestamp: 1712716343937
+     message: "finished to transfer FE type from INIT to LEADER"
+     status: "info"
+     tags:
+       - "source:LOGS_SOURCE"
+     timestamp: 1712716343937
+ -
+   sample: "I0328 10:22:09.303526 1066 olap_server.cpp:844] try to perform path gc by tablet!"
```

Comment:
I have an issue to test this one in staging. There seems to be note date in this log sample, but there is an expected one in the grok parser?

Reply:
39003729303 for the timestamp means it is definitely wrong

Reply:
The date (MMdd) does exist, 'I0328' includes the log level and date information.

During the deployment process of Celerdata, different components, FE/BE, will be deployed on different host machines. This type of log is from the BE component.

Reply:
The issue with the date and timestamp not matching, I think I know the reason now. I have previously rewritten some log data into the logs for testing purposes.
Thank you for your solid review!

Reply:
There is currently a CI issue that requires your assistance to look into. @thibaultkrebs 

Firstly, let me explain how the number `39003729303` came about. According to the parsed content "I0328 10:22:09.303526 1066 olap_server.cpp:844] try to perform path gc by tablet!", the CI validate-logs check expects the number `39003729303`. Clearly, this value does not contain any year information.

I have seen similar configurations on the Datadog website: `https://docs.datadoghq.com/logs/log_configuration/parsing/?tab=matchers#glog-format`. The parsed result includes year information.

The configuration for the Datadog integration is the same as this. The expression used is `{date("MMdd HH:mm:ss.SSSSSS"):timestamp}`. The result I parsed locally does include year information.
![image](https://github.com/DataDog/integrations-extras/assets/7991096/961f1617-b444-45f1-abdb-2f7041a501d0)

Reply:
Yes you are right, I got mislead by believing the number after the I was a code. But this is correct, it is just the date being parsed.

I replicated the grok parser with your sample in Datadog and indeed this is parsing the timestamp well.
![image](https://github.com/DataDog/integrations-extras/assets/150821137/32c745d2-ce8b-4755-912e-3bafc125a82f)

We have a problem with our internal tool to generate this result in the CI. I will need to investigate. Is your integration urgent to be deployed, or if we delay for few days the time I find the issue of fine on your side?

Reply:
I think we can accept to wait few days. During the waiting period, Can you continue to review other parts of this PR?

Reply:
I will give you an update by this week on the log part. For other files included in this PR, some other people will be in charge to review. I invite you to check with the partner team if you think some areas are still pending reviews.

Reply:
I have discussed internally with our team, and it appears that we won't be addressing this issue immediately. To avoid holding up this PR, please go ahead and remove this test for now. We plan to revisit this within a few weeks to resolve the issue and reintroduce the test.

I apologize for any inconvenience, especially after I had requested you to increase the test coverage...

Reply:
OK, thank you for your feedback